scalableminds/webknossos
# Update error codes

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- try different routes and see if the error code matches the expected reason of failure

### Issues:
- fixes #3748 

------
